tasks/build-secrets

```diff
@@ -16,7 +16,9 @@ metadata:
 data:
 EOF
 cd "$BASE/tasks"
-for f in $(find -type f -o -type l); do
+# This intentionally does not capture subdirs like tasks/s3-keys/. As OpenShift secret volumes don't have subdirectories,
+# these need to be created as a separate volume if and when we ever need that.
+for f in $(find -maxdepth 1 -type f -o -type l); do
     printf '  %s: %s\n' "$(echo $f | sed 's!^./!!; s!/!--!g')" "$(base64 --wrap=0 $f)"
```

Comment:
I guess you can remove half of the path mangling seddery here now?

You could also rewrite your find command like so:

  `find * -type f -o -type l -maxdepth 0`

to get rid of the other half of the sed.  That would be a substantial simplification.

Probably also works:

```sh
for f in *; do
  if [ -f $f ]; then
    printf '  %s: %s\n' "$f" "$(base64 --wrap 0 "$f")"
  fi
done
```

(`test(1)` dereferences links for file-type operations)

Reply:
Thanks for pointing out -- I wanted to do that, then it dropped from my  attention. `for f in *` is awkward as it doesn't consider hidden files; same for `find *`. I wanted to move it into a helper, but as we treat the metrics secret special (trim line endings), I don't bother -- this is rather unimportant anyway.

I diffed the before/after output, and it's identical.